lib/edoc/src/edoc.app.src

```diff
@@ -1,26 +1,31 @@
 % This is an -*- erlang -*- file.
 
 {application, edoc,
- [{description, "EDoc"},
+ [
+  {description, "EDoc OTP library"},
```

Comment:
I don't like this description. EDoc is not OTP and is not what you'd think of as a library.

Reply:
Ahh, yes, this is one more thing that slipped in when I imported from the external repo...

Reply:
I see. Just restore it in that case. :-) Looking at descriptions from other OTP apps, they tend to just have a single word or two.

Reply:
I've modified the description. Please let me know, @richcarl, if it's appropriate now or if I should just leave it at "EDoc".

Reply:
That's good, I like it!